lux/action/univariate.py

```diff
@@ -63,7 +63,10 @@ def univariate(ldf, *args):
             ignore_rec_flag = True
     elif data_type_constraint == "nominal":
         possible_attributes = [
-            c for c in ldf.columns if ldf.data_type[c] == "nominal" and c != "Number of Records"
+            c
```

Comment:
This line split is pretty weird and hard to read. Can we fix this and add a comment on what this list of possible_attributes is used for?